fast/stages/02-security/vpc-sc.tf

```diff
@@ -56,7 +57,7 @@ locals {
       access_levels = coalesce(
         try(var.vpc_sc_perimeter_access_levels.dev, null), []
       )
-      resources           = var.vpc_sc_perimeter_projects.dev
+      resources           = try(var.vpc_sc_perimeter_projects.dev, [])
```

Comment:
can't we use local.vpc_sc_perimeter_projects.$ENV here and in L78 + L97? @ludoo

Reply:
yep we should, good catch